README.md

```diff
@@ -2,58 +2,13 @@
 
 Web API project to aggregate Opdex smart contract transactional data providing a restful connection to the underlying protocols.
 
-## Local Environment Setup
+[![Discord](https://img.shields.io/discord/798074378856169482?color=%235865F2&label=Discord&logo=discord&logoColor=%23FFFFFF)](https://discord.gg/PSJr4Ns85k)
 
-### Auth
+### Want to learn more?
+View the [Opdex documentation](https://docs.opdex.com).
 
-Add a bearer token signing key to your configuration, by running `dotnet user-secrets set AuthConfiguration:Opdex:SigningKey ~y0Ur%sEcr3T*k3Y~`. The key can be any UTF-8 string consisting of 16 bytes or more.
+### Run the API yourself.
+See the [local environment setup](local-environment-setup.md) guidelines.
```

Comment:
We can probably remove this and point to https://docs.opdex.com/docs/platform-api-environment-setup instead, what do you think?

Reply:
Agreed, removed and updated with a link to the docs